src/appendices/contributors.md

```diff
@@ -289,6 +289,7 @@ If you contributed to the BIDS ecosystem and your name is not listed, please add
 | Parul Sethi                                          | 📖🔧⚠️💻                               |
 | Patricia Clement                                     | 💬🐛💻📖🔣💡📋🤔📆⚠️📢                 |
 | Patrick Park                                         | 📖💡💬💻                               |
+| Paul Wighton                                         | 📖                                     |
```

Comment:
@pwighton To avoid losing this to a reversion, please update https://github.com/bids-standard/bids-specification/wiki/Recent-Contributors, as this page is auto-generated from a source during the release process.